src/atomate2/vasp/jobs/mp.py

```diff
@@ -0,0 +1,214 @@
+        the "." character which is typically used to denote file extensions. To avoid
+        this, use the ":" character, which will automatically be converted to ".". E.g.
+        ``{"my_file:txt": "contents of the file"}``.
+    """
+
+    name: str = "MP PreRelax"
+    input_set_generator: VaspInputGenerator = field(
+        default_factory=lambda: MPRelaxR2SCANGenerator(
+            user_incar_settings={"EDIFFG": -0.05, "METAGGA": None, "GGA": "PS"}
+        )
+    )
+
+
+@dataclass
+class MPRelaxMaker(BaseVaspMaker):
+    """
+    Maker to create VASP relaxation job using r2SCAN by default.
+
+    Parameters
+    ----------
+    name : str
+        The job name.
+    input_set_generator : .VaspInputGenerator
+        A generator used to make the input set.
+    write_input_set_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.write_vasp_input_set`.
+    copy_vasp_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.copy_vasp_outputs`.
+    run_vasp_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.run_vasp`.
+    task_document_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.TaskDoc.from_directory`.
+    stop_children_kwargs : dict
+        Keyword arguments that will get passed to :obj:`.should_stop_children`.
+    write_additional_data : dict
+        Additional data to write to the current directory. Given as a dict of
+        {filename: data}. Note that if using FireWorks, dictionary keys cannot contain
+        the "." character which is typically used to denote file extensions. To avoid
+        this, use the ":" character, which will automatically be converted to ".". E.g.
+        ``{"my_file:txt": "contents of the file"}``.
+    """
+
+    name: str = "MP Relax"
+    input_set_generator: VaspInputGenerator = field(
+        default_factory=MPRelaxR2SCANGenerator
+    )
+
+    def make(
+        self, structure: Structure, bandgap: float = 0.0, bandgap_tol: float = 1e-4
+    ):
+        """Set correct k-point density, smearing and sigma based on bandgap estimate.
+
+        Parameters
+        ----------
+        structure : pymatgen.Structure
+            The structure to relax.
+        bandgap : float
+            The bandgap of the material in eV. Used to determine the k-point density.
+        bandgap_tol : float
+            The tolerance for the bandgap. If the bandgap is less than this value, the
+            k-point density will be set to 0.22, otherwise it will be set to a value
+            based on the bandgap.
+
+        Returns
+        -------
+        MPRelaxMaker
+            The maker.
+        """
+        self.input_set_generator.config_dict["INCAR"].update(
+            _get_kspacing_params(bandgap, bandgap_tol)
+        )
+
+        return super().make(structure=structure)
+
+
+@dataclass
+class MPStaticMaker(BaseVaspMaker):
+    """
+    Maker to create VASP static job using r2SCAN by default.
```

Comment:
Confused, this says r2SCAN by default by the default input set generator is `StaticSetGenerator` which is not r2SCAN?

Reply:
You're right. Something must not have gotten saved correctly when we were doing the collaborative editing. I'll fix. Thanks!